.github/workflows/docs.yaml

```
@@ -44,10 +44,18 @@ jobs:
       - name: Run Knit
         run: make knit
 
+      - uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)